csm/conf/configure.py

```diff
@@ -110,9 +111,11 @@ async def execute(self, command):
                        f"{e} - {str(traceback.format_exc())}")
             Log.error(err_msg)
             raise CsmSetupError(err_msg)
+        Log.info("csm_setup: configure phase completed.")
```

Comment:
csm_setup -> Setup. Change this at all the places.

configure phase completed. -> Successfully passed Config phase.

Reply:
updated